docs/src/Touchable.doc.js

```diff
@@ -68,6 +68,10 @@ card(
         defaultValue: true,
         href: 'fullHeightWidthExample',
       },
+      {
+        name: 'inline',
+        type: 'boolean',
```

Comment:
boolean props :(

Perhaps this could have `display: 'block' | 'boolean'` with default `block`?

Reply:
Agreed in general, but I was thinking of this more like the `inline` prop on Text. If we may that an enum, is that implying that we'd like to allow users to specify something like `flex` or `inlineBlock` in the future?

Reply:
In this case we should remain consistent with other components. We don't want to add more/new boolean props moving forward but consistency trumps that.

Also, this PR overlaps with #906 where @jackhsu978 also added this `inline` prop + other changes.

Reply:
Ah, I didn't realize Jack was implementing the same thing. 😅 This is no longer urgent for me, so I'm fine with abandoning this PR and waiting for Jack's, unless there's reason to continue with this one.

Reply:
If we do want to continue with this PR: it looks like there's a failing test related to a new CSS class I added. Is there a script that needs to be run after adding a class?

Reply:
I did try to add `inline` support in #906, but @bishwei correctly pointed out that it is not necessary. One can just wrap Touchable with `<Box display="inlineBox">`:

```
<Box display="inlineBox">
  <Touchable>
    ...
  </Touchable>
</Box>
```

Note that `fullWidth` prop in Touchable needs to be true for inline to work, but we don't have to explicitly set it because it is true by default.

Reply:
I'm pretty sure it is still necessary for inline layouts. I was facing this problem yesterday, and any non-inline element child in an inline layout breaks to the next line.

Reply:
Can you give some code example? Here is an live example in 906 PR: https://deploy-preview-906--gestalt.netlify.app/#/Touchable